src/components/TodoItem/index.js

```diff
@@ -20,8 +26,13 @@ function TodoItem({ todo, id }) {
 	const dispatch = useDispatch();
 
 	const handleCompleted = (id) => {
-		dispatch(changeType(id));
-		API.editLisTask(id, { isCompleted: todo.isCompleted });
+		dispatch(callingServer());
+		API.editLisTask(id, { isCompleted: todo.isCompleted })
```

Comment:
!IMPORTANT: DO NOT REQUEST API IN COMPONENT

Reply:
Fixed!